### PR TITLE
updated the readme file

### LINKED ISSUE #6: Adding a README.md file
Right now it's just blank and doesn't give any information, so I would like to make a readme.md file.  Please assign this issue to me

### PR DESCRIPTION
**Pull request by cryptofox17**
# Description

Updated the current README.md file and ensured to include necessary information about the project.

Closes #6 

## Type of change  

- [x] This change requires a documentation update

# Checklist:

- [] My code follows the style guidelines of this project
- [] I have performed a self-review of my own code
- [] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [] I have added tests that prove my fix is effective or that my feature works
- [] Any dependent changes have been merged and published in downstream modules

## Screenshots

After :
Screenshot (74)

Screenshot (75)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,17 +2,68 @@
...
 - Backend - django
 - Database - MongoDb
 
+---
+## Contribution 🙌
+
+This project is completely Open source. Any willing to contribute can take up an issue or raise a new and work on it after getting assigned.
+
+---
+## License ✍
```
Comment (codewithvk) [inline README.md:47]: please remove the License information from here?

**Review by codewithvk — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -2,17 +2,68 @@
 
 # Girlscript Community Website
   <div align="center">
-  <img align="center" src="Assets/GsLogo.PNG" alt="Image of gs" />
+  <img align="center" src="Assets/GsLogo.PNG" alt="Image of gs"/>
```
Comment (codewithvk) [inline README.md:5]: Please use `Assets/GS_logo_black (1).png`.

**Review by codewithvk — CHANGES_REQUESTED** [review]

**Comment by cryptofox17** [comment]
@codewithvk done

**Comment by codewithvk** [comment]
Thanks for PR @cryptofox17

**Pull request merged**